src/TableContent.js

```diff
@@ -3,10 +3,40 @@ import PropTypes from 'prop-types'
 import { Table } from 'evergreen-ui'
 import URI from 'urijs'
 import ReactTooltip from 'react-tooltip'
+import _ from 'lodash'
 
 import tableHeaderCells from './tableHeaderCells'
 import TooltipIcon from './TooltipIcon'
 
+const checkInvalidFiles = async (endpoint, host, checkedRows) => {
+  const url = URI(endpoint, host).search({accession: checkedRows}).toString()
+  try {
+    const response = await fetch(url)
+    if (!response.ok) {
+      throw new Error(`${url} => ${response.status}`)
+    }
+    return await response.json()
+  } catch (e) {
+    return {
+      error: `${e.name}: ${e.message}`,
+      loading: false
+    }
+  }
+}
+
+const alertInvalidFiles = async (endpoint, host, checkedRows) => {
+  const downloadUrl = URI(`experiments/download/zip`, host).search({accession: checkedRows}).toString()
```

Comment:
Move this `const` right above `if (_.isEmpty(data)) {`. As a habit, declare variables and functions as close as possible to where they’re used.

Reply:
Yes. Replaced the `const`.